### DISCUSSION TITLE
【目录】标题中的公式识别不出来

### DISCUSSION THREAD

**Discussion by nowscott** [tangly1024/NotionNext#1977] [Q&A]
**描述bug**
如果标题中有行内公式，就会显示不出来

**截图**
<img width="520" alt="截屏2023-08-15 14 29 30" src="https://github.com/tangly1024/NotionNext/assets/86339284/c12a78b0-03d5-4f1d-a8fe-c18236f79d30">
<img width="187" alt="截屏2023-08-15 14 29 23" src="https://github.com/tangly1024/NotionNext/assets/86339284/e87484dd-03fe-420b-be4a-6976e1809575">

**环境**

- 操作系统: macOS
- 浏览器 chrome
- NotionNext版本 4.0.13
- 主题 next

**Comment by Li-Major** [comment]
现在toc拿到的目录信息，都是已经经过`getTextContent()`处理的
如果把getPageTableOfContents里面的这个方法拿掉
应该会对很多地方造成影响
要不就是类似再增加一个post.rawToc之类的属性？ 
看看@tangly1024 老师怎么考虑
